build/depends.py

```diff
@@ -185,27 +180,19 @@ class Qt(Dependence):
                          'osx': '/Library/Frameworks',
                          'windows': 'C:\\qt\\5.11.1'}
 
-    @staticmethod
-    def qt5_enabled(build):
-        return int(util.get_flags(build.env, 'qt5', 1))
-
     @staticmethod
     def uic(build):
-        qt5 = Qt.qt5_enabled(build)
-        return build.env.Uic5 if qt5 else build.env.Uic4
+        return build.env.Uic5
 
     @staticmethod
-    def find_framework_libdir(qtdir, qt5):
+    def find_framework_libdir(qtdir):
         # Try pkg-config on Linux
         import sys
         if sys.platform.startswith('linux'):
             if any(os.access(os.path.join(path, 'pkg-config'), os.X_OK) for path in os.environ["PATH"].split(os.pathsep)):
                 import subprocess
                 try:
-                    if qt5:
-                        qtcore = "Qt5Core"
-                    else:
-                        qtcore = "QtCore"
+                    qtcore = "Qt5Core"
```

Comment:
This variable is no longer necessary. `qtcore` on the next line can simply be replaced with `Qt5Core`.

Reply:
done